lib/evmone/baseline.cpp

```diff
@@ -19,15 +19,23 @@ CodeAnalysis analyze(const uint8_t* code, size_t code_size)
     static_assert(OP_PUSH32 == std::numeric_limits<int8_t>::max());
 
     CodeAnalysis::JumpdestMap map(code_size);  // Allocate and init bitmap with zeros.
-    for (size_t i = 0; i < code_size; ++i)
+    size_t i = 0;
+    while (i < code_size)
     {
         const auto op = code[i];
         if (static_cast<int8_t>(op) >= OP_PUSH1)  // If any PUSH opcode (see explanation above).
             i += op - size_t{OP_PUSH1 - 1};       // Skip PUSH data.
         else if (INTX_UNLIKELY(op == OP_JUMPDEST))
             map[i] = true;
+        ++i;
     }
-    return CodeAnalysis{std::move(map)};
+
+    // i is the needed code size including the last push data (can be bigger than code_size).
+    std::unique_ptr<uint8_t[]> padded_code{new uint8_t[i + 1]};
```

Comment:
`make_unique` would zero-initialize, as opposed to this?

Reply:
Yes

Reply:
I would maybe add a comment that this leaves it unitialized.